lib/index.d.ts

```diff
@@ -133,6 +133,62 @@ declare namespace Stardog {
             /** Converts a set of integrity constraints into an equivalent SPARQL query for a given database. */
             function convert(conn: Connection, database: string, icvAxioms: string, options: { contentType: HTTP.ContentMimeTypes }, params?: { graphUri: string }): Promise<HTTP.Body>;
         }
+
+        /** Commands that use the reasoning capabilities of a database */
+        namespace reasoning {
+            /**
+             * Provides an explanation for an inference within a transaction
+             * 
+             * @param {Connection} conn the Stardog server connection
+             * @param {string} database the name of the database
+             * @param {string} inference RDF representing the inference to be explained
+             * @param {object} options an object specifying the contentType of the RDF data (e.g., text/turtle)
+             * @param {object} params additional parameters if needed  
+             */
+            function explainInference(conn: Connection, database: string, inference: string, options: { contentType: string }, params?: object);
+
+            /**
+             * Provides the reason why a database is inconsistent, as reported by db.reasoning.consistency
+             * 
+             * @param {Connection} conn the Stardog server connection
+             * @param {string} database the name of the database
+             * @param {object} options an object optionally specifying the URI of a graph to evaluate
+             * @param {object} params additional parameters if needed  
+             */
+            function explainInconsistency(conn: Connection, database: string, options: { namedGraph: string }, params?: object);
```

Comment:
good point- I suppose that's true. if so, we could still do `options: { namedGraph?: string }`